examples/thermomechanics/inputs/thermal_deformation.json

```diff
@@ -1,13 +1,16 @@
 {
-    "num_cells"             : {"value": [100, 30, 3]},
-    "system_size"           : {"value": [1.0, 0.3, 0.03], "unit": "m"},
-    "density"               : {"value": 3980,  "unit": "kg/m^3"},
-    "elastic_modulus"       : {"value": 370e+9, "unit": "Pa"},
-    "thermal_coefficient"   : {"value": 7.5E-6, "unit": "oC^{-1}"},
-    "reference_temperature" : {"value": 20.0, "unit": "oC"},
-    "horizon"               : {"value": 0.03, "unit": "m"},  
-    "final_time"            : {"value": 0.01, "unit": "s"},
-    "timestep"              : {"value": 7.5E-7,  "unit": "s"},
-    "output_frequency"      : {"value": 100},
-    "output_reference"      : {"value": true}
+    "num_cells"               : {"value": [101, 31, 3]},
+    "system_size"             : {"value": [1.0, 0.3, 0.03], "unit": "m"},
+    "density"                 : {"value": 3980,  "unit": "kg/m^3"},
+    "elastic_modulus"         : {"value": 370e+9, "unit": "Pa"},
+    "thermal_expansion_coeff" : {"value": 7.5E-6, "unit": "oC^{-1}"},
+    "thermal_conductivity"    : {"value": 31, "unit": "W/(m.K)"},
```

Comment:
Since we are not solving a heat transfer problems here, do we need the ""thermal_conductivity" and "specific_heat_capacity"?